index.js

```diff
@@ -1,5 +1,3 @@
-'use strict'
```

Comment:
why was this removed?

Reply:
I can fix all of the above. But starting from this point I need to solve two problems.

First one.
When I clone the project, the linter already shows some errors.


`npm install
npm run lint

C:\var\www\html\is-website-vulnerable-master>npm run lint

> is-website-vulnerable@0.0.0-development lint C:\var\www\html\is-website-vulnerable-master
> eslint . && npm run lint:lockfile


C:\var\www\html\is-website-vulnerable-master\bin\is-website-vulnerable.js
   3:1   error  'use strict' is unnecessary when implied strict mode is enabled  strict
  25:9   error  Unexpected console statement                                     no-console
  32:23  error  Expected parentheses around arrow function argument              arrow-parens
  48:7   error  Unexpected console statement                                     no-console
  88:5   error  Unexpected console statement                                     no-console
  89:5   error  Unexpected console statement                                     no-console

C:\var\www\html\is-website-vulnerable-master\index.js
  1:1  error  'use strict' is unnecessary when implied strict mode is enabled  strict

C:\var\www\html\is-website-vulnerable-master\src\Audit.js
  2:1  error  'use strict' is unnecessary when implied strict mode is enabled  strict

C:\var\www\html\is-website-vulnerable-master\src\RenderConsole.js
   3:1   error  'use strict' is unnecessary when implied strict mode is enabled  strict
  16:5   error  Unexpected console statement                                     no-console
  42:52  error  Expected parentheses around arrow function argument              arrow-parens
  76:47  error  Expected parentheses around arrow function argument              arrow-parens

C:\var\www\html\is-website-vulnerable-master\src\RenderJson.js
   3:1   error  'use strict' is unnecessary when implied strict mode is enabled  strict
  15:5   error  Unexpected console statement                                     no-console
  27:52  error  Expected parentheses around arrow function argument              arrow-parens
  53:5   error  Unexpected var, use let or const instead                         no-var
  72:50  error  Expected parentheses around arrow function argument              arrow-parens

C:\var\www\html\is-website-vulnerable-master\src\Utils.js
   4:3   error  Expected method shorthand                            object-shorthand
  14:3   error  Expected method shorthand                            object-shorthand
  21:25  error  Expected parentheses around arrow function argument  arrow-parens
  32:3   error  Expected method shorthand                            object-shorthand
  43:3   error  Expected method shorthand                            object-shorthand

✖ 22 problems (22 errors, 0 warnings)
  16 errors and 0 warnings potentially fixable with the `--fix` option.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! is-website-vulnerable@0.0.0-development lint: `eslint . && npm run lint:lockfile`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the is-website-vulnerable@0.0.0-development lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\User\AppData\Roaming\npm-cache\_logs\2020-01-05T09_46_19_863Z-debug.log
`

Reply:
I believe you may have some global lint and prettier rules, either in your IDE or for the user that override the project in some way. I did a clean git clone && npm install && npm run lint and it passes just alright.

Reply:
I did the same without my IDE and everything works correctly for me! Thanks.
Now I need time to fix everything and I will do a pullrequest.

Reply:
Great! :-)